README.md

```diff
@@ -1,24 +1,68 @@
-# README
+# Transactions App
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> This app helps you track expenses you would like to organize and keep track of. For example, if you are a frequent traveler, you can use this app to create a category for traveling and you can add the cost of each trip to your travel category. The app will keep a total of what all of your expenses are and when you click on a category you will receive a breakdown of each of the expenses.
 
-Things you may want to cover:
+# Live demo
+For a live demo of the project, [click here](https://powerful-beach-62790.herokuapp.com/home).
 
-* Ruby version
+## Getting Started
 
-* System dependencies
+To get a local copy of the project, follow these steps: 
+1. Open the terminal and navigate to the folder where you would like to put these files.
+2. Type the following commands into the terminal: 
+ ```
+ git clone https://github.com/AKeeganDev/ruby_transactions_app.git
+ ```
+ ```
+ cd recipe-app
+ ```
 
-* Configuration
+Check the ruby version being used and install it if it is not the same with that of the local machine
+Set up database direct them to the config/database.yml file
+```
+bundle install
+```
+```
+'npm install' or 'yarn install'
+```
+```
+rails db:create
+```
+```
+rails db:migrate
+```
+```
+rails 'db:seed' if there is data in the seed
+```
+```
+'rails s' to start the server locally
+```
 
-* Database creation
+To execute the tests, run the following command in your terminal:
+```
+rspec spec
+```
 
-* Database initialization
+## Author
 
-* How to run the test suite
+👤 **Aaron Keegan**
 
-* Services (job queues, cache servers, search engines, etc.)
+- GitHub: [@AKeeganDev](https://github.com/AKeeganDev)
 
-* Deployment instructions
+## 🤝 Contributing
 
-* ...
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/jsug9/recipe-app/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
```

Comment:
- [ ] It's a nice and smart thing to give appropriate credit to **[Gregoire Vella](https://www.behance.net/gregoirevella)** on Behance, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.

Reply:
Oh yes, I forgot this. Thank you for checking that. I'll update and resubmit. Thank you!

Reply:
Oops. totally forgot it, thank you!